src/commands/githubActions/bumpUpstream/index.ts

```diff
@@ -99,110 +95,80 @@ async function gaBumpUpstreamHandler({
   await buildAndComment({ dir, commitSha: gitHead.commit, branch: branchName });
 }
 
-async function getUpstreamRepoVersions(upstreamSettings: UpstreamSettings[]): Promise<UpstreamRepoMap | null> {
-
-  const upstreamRepoVersions: UpstreamRepoMap = {};
-
-  try {
-    for (const { arg: upstreamArg, repo: upstreamRepo } of upstreamSettings) {
-
-      const [owner, repo] = upstreamRepo.split("/");
-      const githubRepo = new Github({ owner, repo });
-
-      const releases = await githubRepo.listReleases();
-      const latestRelease = releases[0];
-      if (!latestRelease) throw Error(`No release found for ${upstreamRepo}`);
-
-      const newVersion = latestRelease.tag_name;
-      if (!isValidRelease(newVersion)) {
-        console.log(`This is not a valid release (probably a release candidate) - ${upstreamRepo}: ${newVersion}`);
-        continue;
-      }
-
-      upstreamRepoVersions[upstreamArg] = { repo, repoSlug: upstreamRepo, newVersion };
-      console.log(`Fetch latest version(s) - ${upstreamRepo}: ${newVersion}`);
-    }
-  } catch (e) {
-    console.error("Error fetching upstream repo versions:", e);
-    throw e;
-  }
-
-  if (isEmpty(upstreamRepoVersions)) return null;
-
-  return upstreamRepoVersions;
-}
-
 /**
  * Updates Docker Compose service build arguments with new versions based on `upstreamRepoVersions`.
  * Creates a deep copy of `compose`, modifies build arguments as needed, and writes the updated
  * compose to `dir` if changes occur. Returns an object detailing updates or `null` if no update is needed.
  *
  * @param {string} dir - Directory for the Compose file.
  * @param {Compose} compose - Original Docker Compose configuration.
- * @param {UpstreamRepoMap} upstreamRepoVersions - Mapping of dependencies to their new versions.
- * @return {ComposeVersionsToUpdate | null} - Details of updated versions or null.
+ * @param {UpstreamSettings[]} upstreamSettings - New versions for the Compose services.
  */
-function updateComposeVersions(dir: string, compose: Compose, upstreamRepoVersions: UpstreamRepoMap): ComposeVersionsToUpdate | null {
-  const newCompose = JSON.parse(JSON.stringify(compose)); // Deep copy
-  const versionsToUpdate: ComposeVersionsToUpdate = {};
+function updateComposeVersions(dir: string, compose: Compose, upstreamSettings: UpstreamSettings[]): void {
```

Comment:
I would rename it to `updateComposeUpstreamVersions`

Reply:
Done